### DISCUSSION TITLE
With remote data configuration, how to clear typed query once you select option

### DISCUSSION THREAD

**Discussion by acetinick** [orchidjs/tom-select#137] [General]
In the example provided and also in our environment, tomselect will keep the typed query input after you select an option.

eg.
https://tom-select.js.org/examples/remote/

Is there a way to clear the "ad" after you select option?

<img width="642" alt="Screenshot 2021-07-02 at 16 17 40" src="https://user-images.githubusercontent.com/571649/124288059-08b75a80-db51-11eb-9b74-8ac95088797f.png">

**Comment by oyejorge** [comment]
Why yes there is

```js
new TomSelect('#select-repo2',{
    onItemAdd:function(){
    	this.setTextboxValue();
    	this.refreshOptions();
    }
}
```

**Reply by acetinick** [reply]
Great stuff thanks!
